DataConnectors/CEF/cef_AMA_troubleshoot.py

```diff
@@ -621,39 +640,43 @@ def incoming_logs_validations(self, mock_message=False):
         poll_obj = select.poll()
         poll_obj.register(tcp_dump.stdout, select.POLLIN)
         while (end_seconds - start_seconds) < tcpdump_time_restriction:
-            if mock_message is True:
+            if mock_message is True and mock_message_counter < mock_message_max:
                 # Sending mock messages
                 mock_message_counter += 1
-                self.send_cef_message_local(514, 1)
-            poll_result = poll_obj.poll(0)
+                self.send_message_local(514, 2)
```

Comment:
Why you increase the amount? Also why increate timeout?

Reply:
Fixed. Changed back to the original state.